### ISSUE TITLE
Compilation failure when building main

### ISSUE THREAD

**Issue by boschcrank** [keycloak/keycloak-benchmark#327]
### Describe the bug

Context: 
* Ubuntu 20.04
* OpenJDK 11
* Maven 3.6.3
* Keycloak version is irrelevant, this is a compile error

* Checkout the `main` branch.
* Run `mvn clean install`.

Maven build fails with

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project keycloak-benchmark-dataset: Compilation failure
[ERROR] .../keycloak-benchmark/dataset/src/main/java/org/keycloak/benchmark/dataset/DatasetResourceProvider.java:[95,51] incompatible types: org.keycloak.models.KeycloakUriInfo cannot be converted to javax.ws.rs.core.UriInfo
```

### Version

20.0.5

### Expected behavior

Project builds, jar is created.

### Actual behavior

Project build fails.

### How to Reproduce?

* Checkout the `main` branch.
* Run `mvn clean install`.

**Comment by ahus1** [comment]
Thank you for reporting this. The main branch is compiling against the main branch of Keycloak which switched to Jakarta EE earlier this week. I'll create a 0.9 release branch which will target Keycloak 21.1 and earlier later today.

**Comment by ahus1** [comment]
A branch `0.9-release` is now available that targets Keycloak 21.1 and earlier.

**Comment by ahus1** [comment]
PR available to fix the compile problems in the main branch: #328
